api_reflector/storage.py

```diff
@@ -0,0 +1,16 @@
+"""
+Handle connections to supported k-v storage databases
+"""
+import pickledb
```

Comment:
Im wondering whether to just use redis by default and make it a requirement for this? 🤔  Especially since it will have an api for expiring the key-values out of the box